enumerable.rb

```diff
@@ -0,0 +1,125 @@
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/ModuleLength,
+
+module Enumerable
+  def my_each
+    return to_enum unless block_given?
+
+    n = 0
+    while n < to_a.length
+      yield(self[n]) if is_a?(Array)
+      yield(keys[n], self[keys[n]]) if is_a?(Hash)
+      yield(to_a[n]) if is_a?(Range)
+      n += 1
+    end
+    self
+  end
```

Comment:
- #my_each when a block is given when self is a hash behaves like #each calls the given block once for each element in self
    expected: "[:a, 1][:b, 2][:c, 3][:d, 4][:e, 5]"
    got: "abcde"
Here you need to return the pair key->value, now the method only returns the key, kindly fix it